FlowCrypt.xcodeproj/project.pbxproj

```diff
@@ -165,6 +165,9 @@
 		759739C02833E9E7004867CD /* AppDelegate.swift in Sources */ = {isa = PBXBuildFile; fileRef = D2A1D3B923FD64AB00D626D6 /* AppDelegate.swift */; };
 		75DCE6A02869EBC0003435F1 /* EmptyFolderCellNode.swift in Sources */ = {isa = PBXBuildFile; fileRef = 75DCE69F2869EBC0003435F1 /* EmptyFolderCellNode.swift */; };
 		949ED9422303E3B400530579 /* Colors.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 949ED9412303E3B400530579 /* Colors.xcassets */; };
+		950F27042AB830640041595D /* ContatAddViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 950F27032AB830640041595D /* ContatAddViewController.swift */; };
```

Comment:
`ContatAddViewController` -> `ContactAddViewController`

Reply:
Oopss!
Thanks for your notice